packages/web/README.md

```diff
@@ -1,4 +1,4 @@
-[La version française suit.](#système-de-design---composants-gc)
+[La version française suit.](#composants-de-système-de-design-gc--web)
 
 # GC Design System Components – Web
```

Comment:
Flagging for discussion: should we remove the `- Web` part from the heading since the web package is GC Design System Components and not just another package?

Reply:
Yeah looking at it more, I think this is a good idea.